cluster/state/mutationlegacylock.go

```diff
@@ -85,7 +89,36 @@ func transformLegacyLock(_ Cluster, signed SignedMutation) (Cluster, error) {
 		return Cluster{}, errors.Wrap(err, "verify legacy lock")
 	}
 
-	// TODO(corver): Implement legacy lock transform.
+	lock := signed.Mutation.Data.(lockWrapper)
```

Comment:
nit: check comma ok please, same as line 64-66 in the same file

Reply:
verify function already checks that type is correct, I'll add a comment